test/e2e_node/topology_manager_test.go

```diff
@@ -666,16 +668,21 @@ func setupSRIOVConfigOrFail(f *framework.Framework, configMap *v1.ConfigMap) *sr
 	dpPod, err := f.ClientSet.CoreV1().Pods(metav1.NamespaceSystem).Create(context.TODO(), dp, metav1.CreateOptions{})
 	framework.ExpectNoError(err)
 
+	if err = e2epod.WaitForPodCondition(f.ClientSet, metav1.NamespaceSystem, dp.Name, "Ready", 120*time.Second, testutils.PodRunningReady); err != nil {
+		framework.Logf("SRIOV Pod %v took too long to enter running/ready: %v", dp.Name, err)
+	}
+	framework.ExpectNoError(err)
+
 	sriovResourceName := ""
 	var sriovResourceAmount int64
 	ginkgo.By("Waiting for devices to become available on the local node")
 	gomega.Eventually(func() bool {
 		node := getLocalNode(f)
-		framework.Logf("Node status: %v", node.Status.Capacity)
+		framework.Logf("Node status allocatable: %v", node.Status.Allocatable)
 		sriovResourceName, sriovResourceAmount = findSRIOVResource(node)
-		return sriovResourceAmount > 0
+		return sriovResourceAmount > minSriovResource
```

Comment:
Yes, checking Allocatable instead of Capacity should be the fix

Reply:
Yes, in my local testing, this worked great. The test ran all the way with no failures, and I could definitely see in the logs, that the SRIOV devices took 3-5 seconds before they were "allocatable", whereas before it was almost immediate when checking capacity.